.github/workflows/codeql.yml

```diff
@@ -26,6 +26,11 @@ jobs:
         language: [ go ]
 
     steps:
+      - name: Harden Runner
```

Comment:
this will break if ci workload is run on containers

Reply:
same for all the other harden runner sections

Reply:
e2e test ran, which were on containers, so this issue seems to be resolved, we can go ahead with merge.